src/legacy/core_plugins/kibana/public/management/sections/objects/lib/resolve_saved_objects.js

```diff
@@ -255,7 +254,8 @@ export async function resolveSavedObjects(savedObjects, overwriteAll, services,
         importedObjectCount++;
       }
     } catch (error) {
-      if (error instanceof SavedObjectNotFound) {
+
+      if (error.constructor.name === 'SavedObjectNotFound') {
```

Comment:
Might want to add a null check here with this change.

Reply:
I'll fix it on the follow up PR (which I'm working on ATM)